lib/svrquery/protocol/titanfall/consts.go

```diff
@@ -8,7 +8,7 @@ const (
 	ServerInfoResponse = byte(78)
 
 	// ServerInfoVersion is the version of a info packets.
-	ServerInfoVersion = byte(3)
+	ServerInfoVersion = byte(7)
```

Comment:
Is this backwards compatible and does it need to be?

Reply:
It wont be backwards compatible becouse this is the value used in the request. The server binaries will need to support v7 for this to work when merged.

Reply:
Should we add a new protocol for the new version that way it’s config instead of code

Reply:
Yeah this is a good idea, i'll do this